sonar-project.properties

```diff
@@ -0,0 +1,17 @@
+# Github organization linked to sonarcloud
+sonar.organization==testcontainers
+
+# Project key from sonarcloud dashboard for Github Action, otherwise pick a project key you like
+sonar.projectKey=testcontainers_testcontainers-go
+
+sonar.projectName=testcontainers-go
```

Comment:
I imagine that for modulegen, it could be
```properties
sonar.projectKey=testcontainers_testcontainers-go_modulegen
sonar.projectName=testcontainers-go/modulegen
```
For examples/bigtable it could be
/bigtable
```properties
sonar.projectKey=testcontainers_testcontainers-go_examples_bigtable
sonar.projectName=testcontainers-go/examples/bigtable
```